paynt/parser/drn_parser.py

```diff
@@ -27,7 +27,7 @@ def parse_drn(cls, sketch_path):
                 pomdp_path = sketch_path + '.tmp'
                 state_player_indications = DrnParser.pomdp_from_posmg(sketch_path, pomdp_path)
                 pomdp = DrnParser.read_drn(pomdp_path)
-                explicit_model = payntbind.synthesis.create_posmg(pomdp, state_player_indications)
+                explicit_model = payntbind.synthesis.posmgFromPomdp(pomdp, state_player_indications)
```

Comment:
I think we mostly use the python naming conventions for bound C++ functions? So I would change this to posmg_from_pomdp.